src/pages/WorkspacesList/index.tsx

```diff
@@ -10,41 +10,74 @@
  *   Red Hat, Inc. - initial API and implementation
  */
 
+import {
+  Alert,
+  AlertActionCloseButton,
+  AlertGroup,
+  AlertVariant,
+  Button,
+  PageSection,
+  PageSectionVariants,
+  Text,
+} from '@patternfly/react-core';
+import { Table, TableBody, TableHeader } from '@patternfly/react-table';
+import { History } from 'history';
 import React from 'react';
 import { connect, ConnectedProps } from 'react-redux';
-import { History } from 'history';
-import { Table, TableBody, TableHeader } from '@patternfly/react-table';
-import { Button, PageSection, PageSectionVariants, Text } from '@patternfly/react-core';
-
-import { container } from '../../inversify.config';
-import { AppState } from '../../store';
-import * as WorkspacesStore from '../../store/Workspaces';
 import CheProgress from '../../components/Progress';
+import WorkspaceDeleteAction from '../../components/Workspace/DeleteAction';
 import WorkspaceIndicator from '../../components/Workspace/Indicator';
 import WorkspaceRunAction from '../../components/Workspace/RunAction';
 import WorkspaceStopAction from '../../components/Workspace/StopAction';
-import WorkspaceDeleteAction from '../../components/Workspace/DeleteAction';
-import { Debounce } from '../../services/helpers/debounce';
-import { selectIsLoading, selectAllWorkspacesByName } from '../../store/Workspaces/selectors';
 import { WorkspaceStatus } from '../../services/helpers/types';
+import { AppState } from '../../store';
+import * as WorkspacesStore from '../../store/Workspaces';
+import { selectAllWorkspacesByName, selectIsLoading } from '../../store/Workspaces/selectors';
+import { container } from '../../inversify.config';
+import { Debounce } from '../../services/helpers/debounce';
 
 import './WorkspacesList.styl';
 
 const SECTION_THEME = PageSectionVariants.light;
 
-type Props =
-  MappedProps
-  & {
-    history: History
-  };
+type Props = MappedProps & { history: History };
+
+type State = {
+  alertVisible: boolean;
+  alertMessage: string;
+  alertVariant: AlertVariant;
+};
 
-export class WorkspacesList extends React.PureComponent<Props> {
+export class WorkspacesList extends React.PureComponent<Props, State> {
   private debounce: Debounce;
+  private readonly hideAlert: () => void;
+  private readonly showAlert: (title: string, variant?: AlertVariant) => void;
 
   constructor(props: Props) {
     super(props);
 
+    this.state = {
+      alertVisible: false,
```

Comment:
Please, use `Alert` and `AlertGroup` components in accordance to the documentation.

This implementation of workspaces list is able to show various number of alert notifications and attempt to close one of them will obviously hide all notification.

Reply:
Please, use Alert and Alert Group components in accordance to the documentation. Totally agree Let us discuss

Reply:
My point - it is according to documentation as in other places with show alert

Reply:
I meant that on documentation page there are examples of recommended way to use the `AlertGroup` component. Those examples explicitly suggest to manage alert messages in component's state rather than hide the `AlertGroup` in DOM.

Reply:
We discussed this offline and reached an agreement to implement in the way the documentation examples provide.